DatabaseWorkshop.md

```diff
@@ -0,0 +1,185 @@
+# Adding persistance to ToDoBackend with Swift-Kuery-ORM
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/"> 
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack"> 
+</a>
+</p>
+
+## Prerequisite
+This is a follow on tutorial to our [ToDoBackend tutorial](https://github.com/IBM/ToDoBackend). Please complete that before proceeding with this tutorial.
+
+## Installing PostgreSQL
+
+Before we start, ensure you have PostgreSQL installed and running. In Terminal, run:
+```
+brew services start postgresql
+```
+
+If it's already running then skip ahead to step 1. If you do not have it installed then run the following from Terminal:
+```
+brew install postgresql
+brew services start postgresql
+```
+
+## Persistance over local storage
+Adding a database connection means your state remains after you close your server down, and depending on where your database is stored, different applications on different machines can access the data (for instance, in a Cloud application).
+
+### Step 1 - Create a Database
```

Comment:
We don't use the format Step 1 - etc. in our other tutorials. You can use the format:
### 1. Create the database 
if you like as that's been used in the parent tutorial so would be consistent.

Reply:
✅